docs/application-developer-dev-kubeconifg.md

```diff
@@ -0,0 +1,32 @@
+# Application developer kube-config for development
```

Comment:
Maybe we should change this to "Application developer kubeconfig for contributing" or ".. for contributors".

Since contributors is the word we should use for us who works on compliantkubernets.

Reply:
Sounds good, will update :+1: